cmd/subnetcmd/deploy.go

```diff
@@ -46,7 +46,7 @@ subnet and deploy it on Fuji or Mainnet.`,
 		RunE:         deploySubnet,
 		Args:         cobra.ExactArgs(1),
 	}
-	cmd.Flags().BoolVarP(&deployLocal, "local", "l", false, "deploy to a local network")
+	cmd.Flags().StringVarP(&deployNetwork, "network", "n", "", "network to which to deploy to (one of 'local' or 'fuji')")
```

Comment:
Oh interesting. I was actually thinking we would do `--local`, `--fuji`, and `--mainnet` and then just check that only one flag was selected. We need to use this pattern a few places, so we should make a global decision. Does anyone have a strong opinion? @holisticode @felipemadero

Reply:
I am suggesting here to use the `--network` flag so I stand to that suggestion. However, no strong opinion.

Reply:
Ok refactored to use dedicated flags

Reply:
No strong opinion